### ISSUE TITLE
Doesn't work with go modules

### ISSUE THREAD

**Issue by liminalitythree** [getlantern/systray#80]
I imported systray into a blank application using go 1.11 modules.

I get the error
```
go: finding github.com/getlantern/systray latest
build bitbucket.org/e-liminality/testguistuff: cannot find module for path github.com/getlantern/systray
```

I'm not sure if this is an issue with systray or with go modules. I found a few of these issues that were caused by go modules bugs but they were all fixed.

**Comment by benpye** [comment]
@liminalitythree Did you manage to find a way to get this working?

**Comment by raunaqrox** [comment]
It is working for me with go modules, did you do `export GO111MODULE=on` ?

**Comment by Crosse** [comment]
I'm guessing this would have been closed in 986b75b, when we added modules support (via go.mod/go.sum).
